packages/env/lib/config/config.js

```diff
@@ -68,9 +69,37 @@ module.exports = async function readConfig( configPath ) {
 		md5( configPath )
 	);
 
+	// The specified base configuration from .wp-env.json or from the local
+	// source type which was automatically detected.
+	const baseConfig =
+		( await readRawConfigFile( '.wp-env.json', configPath ) ) ||
+		( await getDefaultBaseConfig( configPath ) );
+
+	// Overriden .wp-env.json on a per-user case.
+	const overrideConfig =
+		( await readRawConfigFile(
+			'.wp-env.override.json',
+			configPath.replace( /\.wp-env\.json$/, '.wp-env.override.json' )
+		) ) || {};
+
+	const detectedLocalConfig =
+		Object.keys( { ...baseConfig, ...overrideConfig } ).length > 0;
+
```

Comment:
The lines above were just moved from lower in the file so that the information is available for the following if statement